src/features/embalm/stepContent/hooks/useCreateSarcohpagus.ts

```diff
@@ -8,6 +8,7 @@ import { setPayloadTxId, setSelectedArchaeologists, setShardsTxId } from 'store/
 import { useDispatch, useSelector } from 'store/index';
 import { Archaeologist } from 'types/index';
 import { useBundlr } from './useBundlr';
+import { useCreateSarcophagus as useEmbalmerFacetCreateSarcophagus } from 'hooks/embalmerFacet';
```

Comment:
My suggestion would be to rename `useCreateSarcophagus` in this file to `useSubmitSarcophagus` since this hook handles taking all the form data and submitting it. I'm open to other suggestions. Then `useEmbalmerFacetCreateSarcophagus` can simply become `useCreateSarcophagus`.